lib/plist.c

```diff
@@ -1436,6 +1447,44 @@ struct stream *prefix_bgp_orf_entry(struct stream *s, struct prefix_list *plist,
 
 	return s;
 }
+int prefix_bgp_rtc_set(char *name, struct prefix *p,
+		       int permit, int set)
+{
+	struct prefix_list *plist;
+	struct prefix_list_entry *pentry;
+
+
+	plist = prefix_list_get(AFI_IP, 0, 1, name);
+	if (!plist) {
+		zlog_info("no prefix list named %s",name);
+		return CMD_WARNING_CONFIG_FAILED;
```

Comment:
definitely, no zlog traces when you push commits. you will save time.

Reply:
Sorry for not removing them and making you so much trouble 😅